ext/com_dotnet/com_olechar.c

```diff
@@ -103,3 +103,58 @@ PHP_COM_DOTNET_API char *php_com_olestring_to_string(OLECHAR *olestring, size_t
 
 	return string;
 }
+
+BSTR php_com_string_to_bstr(zend_string *string, int codepage)
+{
+	BSTR bstr;
```

Comment:
```suggestion
	BSTR bstr = NULL;
```
Otherwise it may be uninitialized if the first `goto fail` is taken.

Reply:
Good catch! Thanks. I have also added the missing `return` in the fail "branch".